instances/atlas/parcellationEntity/Brodmann/BA_cerebralCortex.jsonld

```diff
@@ -8,6 +8,6 @@
     "hasVersion": null,
     "lookupLabel": "BA_cerebralCortex",
     "name": "Brodmann cerebral cortex",
-    "ontologyIdentifier": null,
-    "relatedUBERONTerm": "http://purl.obolibrary.org/obo/UBERON_0000956"
-}
\ No newline at end of file
+    "ontologyIdentifier": "http://purl.obolibrary.org/obo/UBERON_0000956",
+    "relatedUBERONTerm": null
```

Comment:
I think this is too simplified. The UBERON ID is not for Brodmann specifically, which should be the case here. This ID is only related. If you can find "Brodmann cerebral cortex" in an ontology (be it UBERON or something else), then you could link that ID under "ontologyID". If not, I would rather point to the UBERON parcellation under "relatedUBERONTerm".

Reply:
There is no Brodman cerebral cortex. As being part of the UBERON Parcellation itself the cerebral cortex grouping is the one of UBERON.

Question: should we rename it to just cerebral cortex or group them instead under "Brodman area" which has an exact ontology ID?

Reply:
Yes, but "cerebral cortex" in UBERON is not the exact ontology ID for the Brodmann parcellation. Therefore, it cnanot be used as the "ontologyIdentifier" for the Brodmann PE. 

Since we are trying to reflect all atlases as close to reality as possible, the question is: Does "cerebral cortex" exist in the Brodmann terminology? Or is it made up by us to have something to group the PEV under?

1. If it exists, we keep the PE and add "https://openminds.ebrains.eu/instances/UBERONParcellation/cerebralCortex" as "relatedUBERONTerm" (additionally, we could find another ontology that may have the parcellation specifically and add that other ID under "ontologyIdentifier").
2. If something similar exists but not as "cerebral cortex", we should fix the name of the PE to whatever it really is. Additionally, "cerebral cortex" can then be added as an "alternativeName" if it corresponds well. I doubt that this should be "Brodman area" since this is just a grouping term for terms that correspond to Brodmann areas. 
3. If Brodmann does not have anything beyond the areas themselves, then we should remove the "Brodmann_cerebralCortex" all together. If the original parcellation/terminology is flat, then so is our representation in SANDS. 

I hope this makes sense?

Reply:
I definitely want to group them. Then we should do the same as UBERON and just use "Brodman area". I'll do the changes as soon as I can

Reply:
To explain more: even if there is no official Brodman cerebral cortex the Brodman areas are always presented under cerebral cortex which should be also captured in a respective terminology.

Reply:
Maybe we can call it "Brodman cerebral areas" in SANDS linking to the UBERON grouping term "Brodman area"

Reply:
I think that would be fine. "Brodmann cerebral areas" is more descriptive. Just add "Brodmann area" as the "alternativeName" and it should be fine.

Reply:
@UlrikeS91 I thought about it again. I think we should stick to the name "Brodmann cerebral cortex" and link directly to the UBERON "Brodmann area" via the ontology identifier and to the cerebral cortex as related UBERON term. I think this is a special case since Brodmann is integrated into the UBERON ontology.